src/Configuration/AbstractConfiguration.php

```diff
@@ -88,9 +101,6 @@ public function setAllowedOrigins(array $origins): void
         $this->allowedOrigins = $origins;
     }
 
-    /**
-     * @psalm-return list<string>
-     */
```

Comment:
Why are you omitting this? It makes it more specific, which is generally a good thing.

Reply:
its part of the implementing interface, thus not needed in the Implementation.